src/components/core/DpDataTable/DpSelectPageItemCount.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <select
-      id="item-count"
+      :id="selectId"
```

Comment:
Line 15 has to be adapted then, too.

Reply:
thanks for spotting that e16ace514de60e25f9ed320cbc145926a783f5da